src/core/runtime/config-browser.js

```diff
@@ -28,8 +28,8 @@ module.exports = () => ({
   ],
   Swarm: {
     ConnMgr: {
-      LowWater: 600,
-      HighWater: 900
+      LowWater: 200,
+      HighWater: 500
```

Comment:
We can try this, but I think 500 is going to end up being too high for the browser. I think it's more reasonably going to be in the range of 50-200.

Reply:
I'm going to get some testing done for this in the browser and adjust it accordingly, for now just setting it to be the same as Node.js.